src/components/houses details/HouseDetails.js

```diff
@@ -1,77 +1,50 @@
-        <div className="detail-container">
-          <div className="image">
-            <img src={house.image} alt={house.name} />
-          </div>
-          <div className="details">
-            <h5 className="name">{house.name}</h5>
-            <div className="ulIcon">
-              <ul>
-                <li className="first">
-                  <p>City: </p>
-                  <p>{house.city}</p>
-                </li>
-                {/* <li className="first">
-                  <p>State: </p>
-                  <p>{housedetail.state}</p>
-                </li> */}
-                <li className="first">
-                  <p>Daily Rent:</p>
-                  <p>
-                    $
-                    {house.appartment_fee}
-                  </p>
-                </li>
-              </ul>
-              <div className="iconColor">
-                <h4>
-                  DISCOVER MORE COLORS
-                  <AiOutlineRight />
-                </h4>
-                <img src={icon} alt="icon" />
-              </div>
-              <Link to="/reserve">
-                <button type="button" aria-label="Go back" className="reserve-btn">
-                  Book House
-                  <AiOutlineRightCircle />
-                </button>
-              </Link>
-            </div>
+const HousesDetails = () => (
+  <section className="details-sec">
+    <div className="detail-container">
+      <div className="image">
+        <img src={housedetail.image} alt={housedetail.name} />
+      </div>
+      <div className="details">
+        <h5 className="name">{housedetail.name}</h5>
+        <div className="ulIcon">
+          <ul>
+            <li className="first">
+              <p>City: </p>
+              <p>{housedetail.city}</p>
+            </li>
+            <li className="first">
+              <p>State: </p>
+              <p>{housedetail.state}</p>
+            </li>
+            <li className="first">
+              <p>Price:</p>
+              <p>{housedetail.country}</p>
+            </li>
+          </ul>
+          <div className="iconColor">
+            <img src={icon} alt="icon" />
           </div>
+          <Link to="/reserve">
+            <button type="button" aria-label="Go back" className="reserve-btn">
+              Book House
+              <AiOutlineRightCircle />
+            </button>
+          </Link>
         </div>
-      ) : (
-        <div>Loading...</div> // Display a loading message while waiting for API call to complete
-      )}
-      <Link to="/">
-        <button type="button" aria-label="Go back" className="back-btn"><IoCaretBackOutline /></button>
-      </Link>
-    </section>
-  );
-};
-
-HousesDetails.propTypes = {
-  id: PropTypes.string.isRequired,
-}.isRequired;
+      </div>
+    </div>
+    <Link to="/">
+      <button type="button" aria-label="Go back" className="back-btn"><IoCaretBackOutline /></button>
+    </Link>
+  </section>
+);
 
 export default HousesDetails;
```

Comment:
As stated earlier, please do not make changes to the components made by other team members without asking them first. This can cause issues while collaboration.